examples/default.yaml

```diff
@@ -112,9 +111,13 @@ mounts:
   # 🔵 This file: true (only for "/tmp/lima")
   writable: true
 
-# Mount type for above mounts, such as "reverse-sshfs" (from sshocker), "9p" (EXPERIMENTAL, from QEMU’s virtio-9p-pci, aka virtfs),
+# Deny list of mount types for resolving the "default" type.
```

Comment:
This comment is slightly inaccurate, as the setting is not only used to resolve the default mount type, but it also fails template validation when you set `mountType` to an unsupported setting (which is good).

Maybe

```suggestion
# List of mount types not supported by the kernel of this distro.
# Also used to resolve the default mount type when not explicitly specified.
```

Reply:
done